adapter-excel/src/main/java/ch/prevo/open/node/adapter/excel/ExcelConstants.java

```diff
@@ -0,0 +1,53 @@
+package ch.prevo.open.node.adapter.excel;
+
+import java.time.LocalDate;
+import java.time.ZoneId;
+import java.util.Date;
+
+public final class ExcelConstants {
```

Comment:
I really like the idea of extracting all these constants into one class 👍 Do you think you could group them by something, maybe by purpose (input, output, error?) the reason why I'm asking is that I've seen the exact same constants in the notificationwriter, but with different indices which was bit confusing.

Reply:
The indexes differ between input- and output-file. The indexes here are for the input file (and for the error-file, which is based on the input-file).

Reply:
Yeah I was aware of that, but do you think we could kinda group them like with an static inner class? 
ExcelConstants.Input.ROW_HEADER / same for output?

Reply:
Good idea, will do that.